cmd/analysis/leveldb/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+	"path"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+func main() {
+	dataParam := flag.String("data", path.Join(os.Getenv("HOME"), ".darknode/data"), "Data directory")
+
+	orderFragments, err := leveldb.OpenFile(path.Join(*dataParam, "orderFragments"), nil)
+	if err != nil {
+		log.Fatalf("cannot open leveldb: %v", err)
+	}
+
+	iter := orderFragments.NewIterator(nil, nil)
+	fragmentsCount := 0
+	for iter.Next() {
+		// Remember that the contents of the returned slice should not be modified, and
+		// only valid until the next call to Next.
+		key := iter.Key()
+		orderID := common.ToHex(key)
```

Comment:
`ToHex` has the warning `Deprecated: use hexutil.Encode instead` (applies to two other uses of `ToHex`)